### PR TITLE
dir: add German translation

### PR DESCRIPTION
**Pull request by derNiklaas**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.de/windows/dir.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# dir
+
+> Zeigt den Inhalt von Arbeitsverzeichnisses an.
+> Weitere Informationen: <https://docs.microsoft.com/de-de/windows-server/administration/windows-commands/dir>.
```
Comment (navarroaxel) [inline pages.de/windows/dir.md:4]: We prefer language agnostic links for Microsoft pages.
```suggestion
> Weitere Informationen: <https://docs.microsoft.com/windows-server/administration/windows-commands/dir>.
```

File: pages.de/windows/dir.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`dir {{pfad/zu/verzeichnis}} /A`
+
+- Zeigt den Inhalt mit Dateibesitzinformationen des aktuellen Arbeitsverzeichnisses an:
+
+`dir /q`
+
+- Zeigt den Inhalt mit Dateibesitzinformationen des angegebenen Arbeitsverzeichnisses an:
+
+`dir {{pfad/zu/verzeichnis}} /q`
```
Comment (navarroaxel) [inline pages.de/windows/dir.md:28]: We only accept translations of the main pages, to keep this simple to maintain. You can add  these examples to the English page if you want.

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

File: pages.de/windows/dir.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# dir
+
+> Zeigt den Inhalt von Arbeitsverzeichnisses an.
```
Comment (pixelcmtd) [inline pages.de/windows/dir.md:3]:
```suggestion
> Zeige den Inhalt von Verzeichnissen an.
```

File: pages.de/windows/dir.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+- Zeigt den Inhalt des aktuellen Arbeitsverzeichnisses an:
+
+`dir`
+
+- Zeigt den Inhalt des angegebenen Arbeitsverzeichnisses an:
+
+`dir {{pfad/zu/verzeichnis}}`
+
+- Zeigt den Inhalt (inklusive versteckte Inhalte) des aktuellen Arbeitsverzeichnisses an:
```
Comment (pixelcmtd) [inline pages.de/windows/dir.md:14]:
```suggestion
- Zeige den Inhalt, inklusive versteckter Inhalte, des aktuellen Arbeitsverzeichnisses an:
```

File: pages.de/windows/dir.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+`dir /A`
+
+- Zeigt den Inhalt (inklusive versteckte Inhalte) des angegebenen Arbeitsverzeichnisses an:
```
Comment (pixelcmtd) [inline pages.de/windows/dir.md:18]:
```suggestion
- Zeige den Inhalt, inklusive versteckte Inhalte, des angegebenen Verzeichnisses an:
```

File: pages.de/windows/dir.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
+# dir
+
+> Zeigt den Inhalt von Arbeitsverzeichnisses an.
+> Weitere Informationen: <https://docs.microsoft.com/windows-server/administration/windows-commands/dir>.
+
+- Zeigt den Inhalt des aktuellen Arbeitsverzeichnisses an:
+
+`dir`
+
+- Zeigt den Inhalt des angegebenen Arbeitsverzeichnisses an:
```
Comment (pixelcmtd) [inline pages.de/windows/dir.md:10]:
```suggestion
- Zeige den Inhalt des angegebenen Verzeichnisses an:
```

File: pages.de/windows/dir.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
+# dir
+
+> Zeige den Inhalt von Verzeichnissen an.
+> Weitere Informationen: <https://docs.microsoft.com/windows-server/administration/windows-commands/dir>.
+
+- Zeigt den Inhalt des aktuellen Arbeitsverzeichnisses an:
```
Comment (pixelcmtd) [inline pages.de/windows/dir.md:6]:
```suggestion
- Zeige den Inhalt des aktuellen Verzeichnisses an:
```

File: pages.de/windows/dir.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+`dir`
+
+- Zeige den Inhalt des angegebenen Verzeichnisses an:
+
+`dir {{pfad/zu/verzeichnis}}`
+
+- Zeige den Inhalt, inklusive versteckter Inhalte, des aktuellen Arbeitsverzeichnisses an:
```
Comment (pixelcmtd) [inline pages.de/windows/dir.md:14]:
```suggestion
- Zeige den Inhalt, inklusive versteckter Inhalte, des aktuellen Verzeichnisses an:
```

**Review by marchersimon — APPROVED** [review]
Thanks @derNiklaas

**Pull request merged**